openvidu-browser/src/OpenVidu/Session.ts

```diff
@@ -1085,13 +1086,18 @@ export class Session extends EventDispatcher {
                 this.openvidu.closeWs();
             }
 
-            this.stopPublisherStream(reason);
+            logger.info('Leaving Session (refreshTrigger=' + refreshTrigger + ')');
 
-            if (!this.connection.disposed) {
-                // Make Session object dispatch 'sessionDisconnected' event (if it is not already disposed)
-                const sessionDisconnectEvent = new SessionDisconnectedEvent(this, reason);
-                this.ee.emitEvent('sessionDisconnected', [sessionDisconnectEvent]);
-                sessionDisconnectEvent.callDefaultBehavior();
+            if (!refreshTrigger) {
```

Comment:
@sapiri So we will not stop the publisher on disconnect?

Reply:
it is variables that keep on this, after refresh they no longer exist except sending the events, that this what cause the infinity loop